### ISSUE TITLE
Renamed without changing it inside the struct. Intended or not?

### ISSUE THREAD

**Issue by facebamm** [KhronosGroup/Vulkan-Headers#192]
You rename "VkPipelineStageFlags2KHR" to "VkPipelineStageFlagBits2KHR" and "VkAccessFlags2KHR" to "VkAccessFlagBits2KHR" without changing it inside the struct. Is it intended or will it be changed later? 
## For example

https://github.com/KhronosGroup/Vulkan-Headers/blob/e1d8c4931d8e4b6c4a8e633c163ed3aa82d6ac1c/include/vulkan/vulkan_core.h#L7584

 
https://github.com/KhronosGroup/Vulkan-Headers/blob/e1d8c4931d8e4b6c4a8e633c163ed3aa82d6ac1c/include/vulkan/vulkan_core.h#L7672-L7679

**Comment by oddhack** [comment]
TL;DR this is correct and as intended.

The FlagBits types define bitmasks that can be combined into the corresponding Flags type, which is passed in structure members and/or function parameters where multiple bits can potentially be specified. For the 32-bit flag types, we defined the FlagBits as C 'enum' types which allowed some typechecking to take place. For the 64-bit flag types like these two, that was no longer possible because there's no way to reliably and portably define a 64-bit enumerated type in all the compilation environments this header has to support. So the FlagBits types still exist as a notational convenience in the XML, and for consistency with the 32-bit flag types. But they have to be a 64-bit scalar integer type, same as the Flags types they correspond to.
